### DISCUSSION TITLE
[Feature Request]: Remove unused php version from UI

### DISCUSSION THREAD

**Discussion by VladimirFrunza** [beyondcode/herd-community#355] [Ideas]
### Feature Description

Replace Install button with Delete button if PHP already installed. So we as users have an opportunity to remove an unused php version from UI. 

While we dont have an remove button, please describe shortly in comments how to do it manually? which folders should be deleted?

**Comment by buddhaCode** [comment]
You can remove them manually, like so:

- Quit Herd
- Remove the following files and directories
  - `/Users/<username>/Library/Application Support/Herd/bin/php8<Minor Version Number>-fpm`
  - `/Users/<username>/Library/Application Support/Herd/bin/php8<Minor Version Number>`
  - `/Users/<username>/Library/Application Support/Herd/config/php/8<Minor Version Number>/`
  - `/Users/<username>/Library/Application Support/Herd/config/fpm/8.<Minor Version Number>-fpm.conf`
- Start Herd again

For example:
```
rm /Users/arne/Library/Application Support/Herd/bin/php83-fpm
rm /Users/arne/Library/Application Support/Herd/bin/php83
rm -r /Users/arne/Library/Application Support/Herd/config/php/83
rm /Users/arne/Library/Application Support/Herd/config/fpm/8.3-fpm.conf
```

**Comment by mchev** [comment]
Here's a minor enhancement of the @buddhaCode example: simply copy and paste this code, then change the PHP version. It resolves the Application Support path issue and utilizes the user directory variable `$HOME`.

```
rm $HOME/Library/Application\ Support/Herd/bin/php74-fpm &&
rm $HOME/Library/Application\ Support/Herd/bin/php74 &&
rm -r $HOME/Library/Application\ Support/Herd/config/php/74 &&
rm $HOME/Library/Application\ Support/Herd/config/fpm/7.4-fpm.conf
```

I would also love to see a PHP uninstall button on Herd

**Comment by imdadullah12** [comment]
Hi, Please add this feature to remove installed PHP version

**Comment by delta1186** [comment]
Yes this would be a nice option to have in order to streamline the removal process.

**Comment by coendr** [comment]
This would be a realy nice feature to have!
